client/src/components/LabeledTable.tsx

```diff
@@ -1,15 +1,19 @@
 import PropTypes from "prop-types";
 import "./LabeledTable.scss";
 
-export const LabeledTable = ({ title, TitleComponent, contents }) => (
-  <section className="labeled-table" aria-label={title}>
-    {title && (
+interface LabeledTableProps {
+  title: string;
+  contents: any;
+  TitleComponent?: any;
+}
+
+export const LabeledTable = (props: LabeledTableProps) => (
```

Comment:
It's possible to still use destructured function args via something like 
```typescript
interface OptionalMiddleName {
    firstName: string;
    middleName?: string;
    lastName: string;
}
function Component({firstName, middleName = "N/A", lastName}:OptionalMiddleName){
    // If middleName wasn't passed in, value will be "N/A"
}
```
No strong opinion one way or the other, but this could cut down the amount of `props.` we need to add

Reply:
Ah yeah I think this is better